_pages/fr/experiments/test-pilot.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title:  "Install Test Pilot"
+permalink: /lang/experiments/test-pilot
```

Comment:
since this is the FR version, please already replace "lang" with "fr" (same for the lang: property below)